src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/direct_query_table.scss

```diff
@@ -0,0 +1,15 @@
+.direct-query-expanded-table thead {
+  display: none;
+}
+
+[id^="row"][id$="expansion"] > td:first-child > div:first-child {
```

Comment:
```suggestion
.direct-query-expanded-table [id^="row"][id$="expansion"] > td:first-child > div:first-child {
```

Reply:
It will introduce many unexpected padding in this case if I restrict to the direct-query-expanded-table ...
<img width="1258" alt="image" src="https://github.com/user-attachments/assets/9d18c101-51ca-4e9e-b048-12ea2cc89496">

Reply:
Fixed. See new commit.